docs/conf.py

```diff
@@ -55,10 +74,8 @@
 # built documents.
 #
 
-from pkg_resources import get_distribution
-release = get_distribution('winton_kafka_streams').version
-# for example take major/minor
-version = '.'.join(release.split('.')[:2])
+from setuptools_scm import get_version
+version = release = get_version(root='..')
```

Comment:
Nice, setuptools_scm is very convenient!

Reply:
Yes, though strictly speaking this change is against their usage advice. See the section “Usage from Sphinx” here: https://github.com/pypa/setuptools_scm/blob/master/README.rst 

The problem is that we are not either doing an editable or real install with pip as the Confluent Kafka package must be built and that doesn’t work cleanly on RTD (this should also help explain why the Kafka related modules are mocked above).